src/legacy/core_plugins/kbn_vislib_vis_types/public/components/options/metrics_axes/custom_extents_options.tsx

```diff
@@ -38,29 +38,25 @@ function CustomExtentsOptions({
   setValueAxis,
   setValueAxisScale,
 }: CustomExtentsOptionsProps) {
-  const [isBoundsMarginValid, setIsBoundsMarginValid] = useState(true);
   const invalidBoundsMarginMessage = i18n.translate(
     'kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBounds.minNeededBoundsMargin',
     { defaultMessage: 'Bounds margin must be greater than or equal to 0.' }
   );
 
-  const setBoundsMargin = useCallback(
-    (paramName: 'boundsMargin', value: number | '') => {
-      const isValid = value === '' ? true : value >= 0;
-      setIsBoundsMarginValid(isValid);
-      setMultipleValidity('boundsMargin', isValid);
+  const isBoundsMarginValid =
+    !axis.scale.defaultYExtents || !axis.scale.boundsMargin || axis.scale.boundsMargin >= 0;
 
-      setValueAxisScale(paramName, value);
-    },
-    [setMultipleValidity, setValueAxisScale]
+  const setBoundsMargin = useCallback(
+    (paramName: 'boundsMargin', value: number | '') =>
+      setValueAxisScale(paramName, value === '' ? undefined : value),
```

Comment:
Looks like it would be nice to use new `NumberInputOption` -> #48117 component here instead

Reply:
Sure, I'll update the code once that PR is merged.